styles/google/catppuccin.user.css

```diff
@@ -0,0 +1,1093 @@
+/* ==UserStyle==
+@name           Google Catppuccin Less
+@namespace      github.com/GitMuslim/userstyles/styles/google
+@homepageURL    https://github.com/GitMuslim/userstyles/tree/feat/google/styles/google
+@version        1.0.3
+@updateURL      https://github.com/GitMuslim/userstyles/raw/feat/google/styles/google/catppuccin.user.css
```

Comment:
Should be `https://github.com/catppuccin/userstyles/raw/main/styles/google/catppuccin.user.css`.